integrations/transportation/rejsekort.js

```diff
@@ -298,7 +347,6 @@ const config = {
   isPrivate: true,
   signupLink: 'https://selvbetjening.rejsekort.dk/CWS/CustomerRegistration/ValidateNemId',
   contributors: ['tranberg'],
```

Comment:
I think you should add your name here :)

Reply:
Okay then. ;)